PocketKit/Sources/PocketKit/MyList/ItemsList/ItemsListItemCell.swift

```diff
@@ -241,11 +202,24 @@ extension ItemsListItemCell {
 
         titleLabel.attributedText = state.model?.attributedTitle
         detailLabel.attributedText = state.model?.attributedDetail
-        favoriteButton.configuration?.image = state.model?.favoriteButtonImage
-        favoriteButton.accessibilityLabel = state.model?.favoriteButtonAccessibilityLabel
 
-        thumbnailView.image = nil
+        favoriteButton.accessibilityLabel = state.model?.favoriteAction?.title
+        favoriteButton.accessibilityIdentifier = state.model?.favoriteAction?.accessibilityIdentifier
+        favoriteButton.configuration?.image = state.model?.favoriteAction?.image
+
+        if let favoriteAction = UIAction(state.model?.favoriteAction) {
+            favoriteButton.addAction(favoriteAction, for: .primaryActionTriggered)
```

Comment:
Do we need to remove all actions before adding an action? I just want to verify that we won't be stacking actions for the same control event.

Reply:
From Apple's docs:

> UIActions are uniqued based on their identifier, and subsequent actions with the same identifier replace previously added actions.

So I take that to mean that we are safe here. This is why the `.favorite()` and `.unfavorite()` helpers use the same identifier.